client/my-sites/site-settings/disconnect-site/confirm.jsx

```diff
@@ -0,0 +1,51 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { flowRight } from 'lodash';
+import { localize } from 'i18n-calypso';
+/**
+ * Internal dependencies
+ */
+import DisconnectJetpack from 'blocks/disconnect-jetpack';
+import DocumentHead from 'components/data/document-head';
+import FormattedHeader from 'components/formatted-header';
+import Main from 'components/main';
+import Placeholder from 'my-sites/site-settings/placeholder';
+import redirectNonJetpack from 'my-sites/site-settings/redirect-non-jetpack';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+
+const ConfirmDisconnection = ( { siteId, siteSlug, translate } ) => {
+	if ( ! siteId ) {
+		return <Placeholder />;
+	}
+
+	return (
+		<Main className="disconnect-site__confirm">
+			<DocumentHead title={ translate( 'Site Settings' ) } />
+			<FormattedHeader
```

Comment:
This is rather a design comment, but I was just thinking: don't we have just too many headings/subheadings here?

![](https://cldup.com/S_0e_uDeQw.png)

cc @keoshi

Reply:
@tyxla we totally do, this should be adjusted.

Reply:
I found this mockup over in the PR desc for #18039

![screen shot 2017-09-13 at 20 30 59](https://user-images.githubusercontent.com/13561163/30438841-2913483e-996a-11e7-99fc-17755fad9e74.png)

But it diverges quite a bit from the dialog, especially regarding the layout of the "By disconnection..." part...

Reply:
Also, I guess we want full-screen (i.e. no sidebar)?

Reply:
@ockham I would say let's leave the big paragraph as is (without the red icon) and just remove the heading. Also, yes to full-screen!